obspy/clients/filesystem/tsindex.py

```diff
@@ -1174,7 +1175,8 @@ def _is_index_cmd_installed(self):
         """
         Checks if the index command (e.g. mseedindex) is installed.
 
-        :raises OSError: If the index command is not installed.
+        :rtype: bool
+        :returns: Returns True the index_cmd is installed.
```

Comment:
Backticks for builtins/variable names, please (for proper formatting in docs):  -> `Returns ``True`` if ``index_cmd`` is installed.`